components/managed-identity/managed-identity.tf

```diff
@@ -77,32 +75,47 @@ locals {
 
   # MIs for managed-identities-sandbox-rg etc - for workload identity with ASO
   mi_cft = {
+    # DCD-CNP-Sandbox
     sbox = {
-      scope = "/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783"
-    }
-    perftest = {
-      scope = "/subscriptions/7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c"
+      subscription_scope  = "/subscriptions/${local.mi_cft[var.env].subscription_id}"
```

Comment:
i meant remove this fully as it shouldn't be needed anymore, likely it can just be inlined to wherever uses it